### PR TITLE
Upgrade Joda-Time to 2.7 and update MANIFEST to accept [2,3)

### PR DESCRIPTION
**Pull request by sagarsane**
For AEM 6.1 and 6.2 compatibility ..

.. Needs to be tested on both AEM 6.1 and AEM 6.2 .. 

I had to also make a minor change in `grabbit.sh` for CURL to work correctly with AEM 6.2

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/TWCable/grabbit/pull/140" target="_blank">ReviewNinja

File: grabbit.sh [hunk 1]
```diff
@@ -30,7 +30,7 @@ function newGrabbitRequest() {
     echo $SET_YELLOW
     echo "Processing....."
     echo $SET_NO_COLOR
-    curl -s -f -X PUT --data-binary "@$configpath" -u $username:$password -D /tmp/grabbit_headers $client$GRABBIT_JOB > /tmp/grabbit
+    curl -s -f -X PUT --header "Content-Type: text/plain" --data-binary "@$configpath" -u $username:$password -D /tmp/grabbit_headers $client$GRABBIT_JOB > /tmp/grabbit
```
Comment (sagarsane) [inline grabbit.sh]: This seems necessary to correctly `PUT` the config file to AEM 6.2 correctly. Without this, in AEM 6.2... the content of the file gets wrongly placed into weirdly managed Request Parameters instead.

This continues to work as it is on AEM 6.1.

**Comment by sagarsane** [comment]
This is ready for testing and review @jdigger @jbornemann @viveksachdeva 

cc @masroormohammed @jazeren1 

The best way to test this would be to use a AEM 6.1 author as server and AEM 6.2 as client and try to copy content from AEM 6.1 author to AEM 6.2 publish.

File: grabbit.sh (hunk 1 above)
Comment (jbornemann) [inline grabbit.sh]: weird..

File: gradle.properties [hunk 2]
```diff
@@ -5,7 +5,7 @@ org.gradle.jvmargs=-XX:MaxPermSize=512m -XX:+CMSClassUnloadingEnabled -XX:+CMSPe
 bundleInstallRoot = /apps/grabbit/install
 
 group = com.twcable.grabbit
-version = 6.0.0
```
Comment (jbornemann) [inline gradle.properties]: Should we keep this as a seperate commit? Makes a bit easier to view RC

Comment (sagarsane) [inline gradle.properties]: Oh yeah for sure. I normally just have it for testing. Will be removing it from this commit before TM.

Comment (jbornemann) [inline gradle.properties]: Oh, gotcha

File: grabbit.sh (hunk 1 above)
Comment (jbornemann) [inline grabbit.sh]: We may need to do something similar for https://github.com/TWCable/grabbit-cli/blob/master/src/main/java/com/twcable/grabbit/tools/jobstarter/JobStarter.java#L148 for 6.2 support

Comment (sagarsane) [inline grabbit.sh]: Yeah true .. or at least test it (may be `application/json` will work there)

**Comment by viveksachdeva** [comment]
@sagarsane : Tested with 6.2 author to 6.1 author.. Worked alright...

**Comment by jbornemann** [comment]
Looks good on 6.1 author and pub +1

**Comment by sagarsane** [comment]
Actually, I think this should also be tested on 6.2 publisher

**Comment by sagarsane** [comment]
Tested fine on AEM 6.1 -> AEM 6.2 publisher as well.

**Comment by jbornemann** [comment]
+1

**Pull request merged**